release-notes/dashboards-observability.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,41 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
+
+- [Feature] Match discover look and feel by @opensearch-trigger-bot in ([#1179](https://github.com/opensearch-project/dashboards-observability/pull/1179)
```

Comment:
Feature? In a PATCH release?

https://semver.org/#spec-item-6
> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backward compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

Have not all non-bugfix backports been revert?

Reply:
Looks like they have an the release notes were fixed in https://github.com/opensearch-project/dashboards-observability/pull/1273. This was an oversight. There's also some confusion on why this is under a "refactor" section, but either way it got reverted from the patch release.